Updating Persisted Query Configuration (#3175)
apollographql/apollo-ios: docs/source/code-generation/codegen-configuration.mdx

			"MyFragment"
		]
	},

>     "operationDocumentFormat" : ["definition", "operationId"],

Reviewer: I think the indentation got messed up here.

</MultiCodeBlock>

For more details, see the section on [downloading a schema](./downloading-schema).

## Operation Manifest Configuration

> Optional settings used to configure the usage of [Persisted Queries or Automatic Persisted Queries](./../fetching/persisted-queries) in order to generate the operation identfier manifest and determine how operation files are generated.

Reviewer: ```suggestion
Optional settings used to configure generation of the operation identifier manifest for use with [Persisted Queries](./../fetching/persisted-queries).
```

The properties of the Operation Manifest configuration object are:

| Property Name | Description |
| ----- | ----------- |
| `path` | Local path where the generated operation manifest file should be written. |
| [`version`](https://www.apollographql.com/docs/ios/docc/documentation/apollocodegenlib/apollocodegenconfiguration/operationmanifestfileoutput/version) | The version format to use when generating the operation manifest. |

<MultiCodeBlock>

```json title="CLI Configuration JSON"
"output": {
    "operationManifest" : {
        "path" : "./generated/operationIdentifiers.json",
        "version" : "persistedQueries"
    }
}
```

```swift title="Swift Codegen Setup"
let configuration = ApolloCodegenConfiguration(
    // Other properties not shown
    output: ApolloCodegenConfiguration.FileOutput(
        operationManifest: .init(
            path: "./generated/operationIdentifiers.json",
            version: .persistedQueries
        )
    )
)
```

> </MultiCodeBlock>

Reviewer: I think this section needs to be deleted? It's an older version of whats directly above it.
